Makefile

```diff
@@ -35,3 +37,29 @@ clean:
 
 requirements.txt: setup.py
 	pip-compile setup.py
+
+.PHONY: scan clean_scan
+scan: clean_scan
```

Comment:
I think it would be better to add this to GitHub actions where it will be executed in clean isolated environment.

Reply:
I'm not sure if we can, at this point. I'm using `images.paas.redhat.com/alm/sonar-scanner` as image. I'm not sure if it's visible externally. The same for the server `https://sonarqube.corp.redhat.com`.